docker-compose.yml

```diff
@@ -0,0 +1,31 @@
+version: '3'
```

Comment:
In production, we will not use docker-compose, so this might be dropped. Nonetheless, if useful for testing, feel free to ignore this comment.